### ISSUE TITLE
Pin allocations in Simulator are for DIO0 and DIO1 

### ISSUE THREAD

**Issue by StuartsProjects** [Bambofy/FOSSASAT-1#11]
The pin allocations for the simulator are;

#define RADIO_DIO0                                      2
#define RADIO_DIO1                                      3
#define RADIO_BUSY                                      9

When using an SX126x, do you need to connect its DIO1 pin to the pin defined as  RADIO_DIO0 ?

**Comment by jgromes** [comment]
That's correct, it is written that way because of SX1278 (and because SX1268 starts at DIO1). However, support for SX1278 will be dropped from the simulator anyway (because of the sync word issue, #10).
